<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ServerConsoleWindow</class>
    <widget class="QMainWindow" name="ServerConsoleWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>960</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Server Console</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                    <widget class="QGroupBox" name="groupBoxStatus">
                        <property name="title">
                            <string>Server Status</string>
                        </property>
                        <layout class="QGridLayout" name="gridLayout_2">
                            <item row="0" column="0">
                                <widget class="QLabel" name="labelStatusCaption">
                                    <property name="text">
                                        <string>Status:</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QLabel" name="labelStatusValue">
                                    <property name="text">
                                        <string>Offline</string>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">color: rgb(200, 50, 50); font-weight: bold;</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="labelConnectionCount">
                                    <property name="text">
                                        <string>Active Connections:</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QLabel" name="labelConnectionCountValue">
                                    <property name="text">
                                        <string>0</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="labelUptime">
                                    <property name="text">
                                        <string>Uptime:</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QLabel" name="labelUptimeValue">
                                    <property name="text">
                                        <string>00:00:00</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item row="1" column="0">
                    <widget class="QGroupBox" name="groupBoxFilters">
                        <property name="title">
                            <string>Filters</string>
                        </property>
                        <layout class="QGridLayout" name="gridLayout_3">
                            <item row="0" column="0">
                                <widget class="QLabel" name="labelSearch">
                                    <property name="text">
                                        <string>Search:</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QLineEdit" name="lineEditFilterText">
                                    <property name="placeholderText">
                                        <string>username, command, status code …</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="2">
                                <widget class="QComboBox" name="comboBoxCommandFilter">
                                    <item>
                                        <property name="text">
                                            <string>All Commands</string>
                                        </property>
                                    </item>
                                </widget>
                            </item>
                            <item row="0" column="3">
                                <widget class="QComboBox" name="comboBoxStatusFilter">
                                    <item>
                                        <property name="text">
                                            <string>All Statuses</string>
                                        </property>
                                    </item>
                                </widget>
                            </item>
                            <item row="0" column="4">
                                <widget class="QPushButton" name="pushButtonClearFilters">
                                    <property name="text">
                                        <string>Clear</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item row="2" column="0">
                    <widget class="QGroupBox" name="groupBoxLogs">
                        <property name="title">
                            <string>Request Log</string>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QTableView" name="tableViewLogs">
                                    <property name="alternatingRowColors">
                                        <bool>true</bool>
                                    </property>
                                    <property name="selectionBehavior">
                                        <enum>QAbstractItemView::SelectRows</enum>
                                    </property>
                                    <property name="selectionMode">
                                        <enum>QAbstractItemView::SingleSelection</enum>
                                    </property>
                                    <property name="horizontalScrollMode">
                                        <enum>QAbstractItemView::ScrollPerPixel</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout">
                                    <item>
                                        <widget class="QPushButton" name="pushButtonExport">
                                            <property name="text">
                                                <string>Export Logs…</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="pushButtonClearLog">
                                            <property name="text">
                                                <string>Clear Log</string>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">color: rgb(180, 50, 50);</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <spacer name="horizontalSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="pushButtonPause">
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                            <property name="text">
                                                <string>Pause Stream</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item row="3" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayoutBottom">
                        <item>
                            <widget class="QPushButton" name="pushButtonShowAdQueue">
                                <property name="text">
                                    <string>Ads Section (Admin)</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pushButtonShowStats">
                                <property name="text">
                                    <string>Statistics</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacerBottom">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pushButtonQuit">
                                <property name="text">
                                    <string>Quit Server</string>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true">color: rgb(200, 50, 50); font-weight: bold;</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
